challenge_5/scss/main.scss

```diff
@@ -0,0 +1,41 @@
+@import "../scss/variables";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  align-items: center;
+  background: $main-color;
+  display: flex;
+  flex-direction: column;
+  height: 600px;
+  justify-content: center;
+  position: relative;
+  width: 800px;
+
+
+  .ice-cream {
+    background: $secondary-color;
+    border-radius: 100px 100px 41px 41px;
+    height: 300px;
+    width: 200px;
+  }
+
+  .ice-cream-stick {
```

Comment:
This could be nested inside the `.ice-cream` class, even if they aren't in the html.

Reply:
57fb396